app/backend2/obj/backend2.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,64 @@
+{
+  "format": 1,
+  "restore": {
+    "/Users/neumanns/Projects/kubecon-eu-2023-opentelemetry-kubernetes-tutorial/app/backend2/backend2.csproj": {}
```

Comment:
Is this supposed to be committed?

Reply:
I don't know? I have no idea what I am doing with dotnet :D